uniffi_bindgen/src/lib.rs

```diff
@@ -161,11 +161,7 @@ fn ensure_versions_compatibility(
             metadata_cmd.manifest_path(p);
         }
         None => {
-            metadata_cmd.current_dir(
-                idl_file
-                    .parent()
-                    .ok_or_else(|| anyhow!("IDL file has no parent folder!"))?,
```

Comment:
This was incorrectly setting the current directory to `./src` rather than the crate root, but `cargo metadata` was walking the directory tree to find the closest enclosing `Cargo.toml` file, so it didn't cause a bug in practice.